background.ts

```diff
@@ -418,7 +418,6 @@ function writeSettings(arg: RecursivePartial<settings.Settings>) {
   _.merge(cfg, arg);
   settings.save(cfg);
   mainEvents.emit('settings-update', cfg);
```

Comment:
Dumb question: do we no longer need to trigger the actions that happen with the `k8s-restart-required` event?

Reply:
Correct; that was used for the old prefs UI (for a notification), which has been removed.